README.md

```diff
@@ -44,16 +44,17 @@ disclosure reporting.
   the events database. In LOCKDOWN mode, only listed binaries are allowed to
   run.
 
-* Event logging: When the system extension is loaded, all binary launches are logged. When in either mode, all unknown or denied binaries are stored in the database to enable later aggregation.
+* Event logging: When the system extension is loaded, all binary launches are
+  logged. When in either mode, all unknown or denied binaries are stored in the
+  database to enable later aggregation.
 
-* Certificate-based rules, with override levels: Instead of relying on a
+* Signature-based rules, with override levels: Instead of relying on a
```

Comment:
Nit: Might want to call these code signing based rules as signature-based rules often means pattern based or heuristic based.

Reply:
Good call, done